doc/general/cash-register-integration/cash-register-integration-failure-scenarios.md

```diff
@@ -0,0 +1,32 @@
+### Failure Scenarios
+This Chapter describes the different scenarios of failure when using the fiskaltrust.Middleware and how to handle them.
+
+#### Failure of the fiskaltrust.Middleware
+If a cash register cannot communicate with the fiskaltrust.Middleware it is most likely due to a failure of the fiskaltrust Middleware. Such a failure means that the electronic recording system is not operational and there is no access to the appropriate journal.
+
+![](./images/07-no-middleware-connection.png)
+
+In this case, the following steps must be taken:
+
+  - The cash register or input station must automatically produce a receipt and its copy.
+  - The receipt must be marked with the identification "electronic recording system failed" and with the current failure counter.
+  - This copy needs to be kept until the failure is resolved. The creation and storing of the receipt copy can also be done electronically by the cash register or terminal.
+  - After re-establishing the communication to fiskaltrust.Middleware, the cash register or the input station must send all receipts marked with the identification "receipt copy, electronic recording system failed" to fiskaltrust.Middleware. The ReceiptCase must be flagged with the code "failed receipt" in order to indicate the failure to fiskaltrust.Middleware, which will then issue a receipt response with the the ftState "Late Signing Mode".
```

Comment:
isn't this process started/ended with a zero receipt (or somethingth like that ^^ I might misremember)?

Reply:
ignore that ^^ its mentioned in line 20